x-pack/plugins/apm/server/lib/metrics/by_agent/default.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Setup } from '../../helpers/setup_request';
+import { getCPUChartData } from './shared/cpu';
+import { getMemoryChartData } from './shared/memory';
+import { PromiseReturnType } from '../../../../typings/common';
+
+export type DefaultMetricsChartsResponse = PromiseReturnType<
+  typeof getDefaultMetricsCharts
+>;
+
+export async function getDefaultMetricsCharts(
+  setup: Setup,
+  serviceName: string
+) {
+  const charts = await Promise.all([
+    getCPUChartData(setup, serviceName),
+    getMemoryChartData(setup, serviceName)
+  ]);
+
+  return { charts };
```

Comment:
Any reason for not just returning `charts`?
```ts
return charts;
```

Reply:
We could! I like API endpoints to return an object so that it's easier to later add data to the result payload which you can't do if the top level item is just an array, so I've been return `{ charts }` as the metrics charts API for now.

Reply:
Okay, that makes sense. Then just leave it like this 👍